api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>Definitely a minor point. But is it the first_data_point or last_data_point since we are taking from the end of the array?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>its the most recent data point! that might be confusing but ill change it to that</p>